pkg/plugins/builtin/ssh/README.md

```diff
@@ -42,15 +42,52 @@ action:
     ssh_key: '{{.config.mySSHKey}}'
 ```
 
-The output resulting from this configuration will be:
+## Requirements
+
+None by default. Ssh credentials should be retrieved from `{{.config.[sshkey]}}` rather than hardcoded on the template.
+
+## Note
+
+The plugin returns two objects, the `Payload` of the execution as defined in the result field of configuration:
 
-```js
+```json
 {
-  "pid": "1234",
-  "uptime": "876123"
+  "pid": "3931",
+  "uptime": "1715606"
 }
 ```
 
-## Requirements
+The `Metadata` to fetch informations about plugin execution:
+
+```json
+{
+  "output": "Connecting...
```

Comment:
nitpicking: I'm afraid this is not valid json :) you're missing`\n` at the end of each line

Reply:
also it looks a bit weird without the cmds, should we give an example where the script starts with set -x ?

Reply:
@loopfz You mean by cmds commands that retrieves pid,service_name... ?